util/observable/action/action.go

```diff
@@ -21,15 +21,6 @@ const (
 	// Replace replaces the current value of the subject with the object.
 	Replace Action = "replace"
 
-	// Prepend prepends the object to the list of values of the subject.
-	Prepend Action = "prepend"
-
-	// Append appends the object to the list of values of the subject.
-	Append Action = "append"
-
-	// Remove removes the object from the list of values of the subject.
-	Remove Action = "remove"
```

Comment:
I assume these are unused by the backend?

Reply:
As far as I checked and afaik, but I might miss something, could you check the backend as well?